sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -2750,4 +2750,26 @@ private[sql] object QueryExecutionErrors extends QueryErrorsBase {
       errorClass = "UNSUPPORTED_EMPTY_LOCATION",
       messageParameters = Map.empty)
   }
+
+  def locationAlreadyExistsInCreate(
+      tableId: TableIdentifier,
+      location: Path): Throwable = {
+    new SparkRuntimeException(
+      errorClass = "LOCATION_ALREADY_EXISTS.CREATE_MANAGED_TABLE",
+      messageParameters = Map(
+        "location" -> toSQLValue(location.toString, StringType),
+        "table" -> toSQLId(tableId.nameParts)))
+  }
+
+  def locationAlreadyExistsInRename(
+      tableId: TableIdentifier,
+      newTableId: TableIdentifier,
+      location: Path): Throwable = {
+    new SparkRuntimeException(
+      errorClass = "LOCATION_ALREADY_EXISTS.RENAME_MANAGED_TABLE",
+      messageParameters = Map(
+        "location" -> toSQLValue(location.toString, StringType),
+        "table" -> toSQLId(tableId.nameParts),
```

Comment:
No longer print possible `catalog` and `database`?

Reply:
It does, see the modified tests.

Reply:
Got it ~